Configs/.config/hypr/scripts/swwwallselect.sh

```diff
@@ -8,8 +8,10 @@ WallPath="$HOME/.config/swww/$gtkTheme"
 
 
 # scale for monitor x res
-x_monres=`cat /sys/class/drm/*/modes | head -1 | cut -d 'x' -f 1`
-x_monres=$(( x_monres*17/100 ))
+x_monres=$(hyprctl -j monitors | jq '.[] | select(.focused==true) | .width')
+monitor_scale=$(hyprctl -j monitors | jq '.[] | select (.focused == true) | .scale' | sed 's/\.//')
+x_monres=$((x_monres * 100 / monitor_scale ))
+x_monres=$(( x_monres*18/100 ))
```

Comment:
same here as well!
also the percentage seems to be changed from `17` to `18` here.